### PR TITLE
Add min/max temperature watch face

### PR DESCRIPTION
**Pull request by MarkBlyth**
The new face is similar to the temperature logging face, only instead it measures the temperature once a minute, and records the high and low for that hour. Only the highest and lowest temperatures from the last 24h are displayed on the screen, creating a nice simple temperature-range interface.

### CODE REVIEW

File: movement/watch_faces/sensor/minmax_face.c [hunk 1]
```diff
@@ -0,0 +1,155 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2023 <#author_name#>
```
Comment (WesleyAC) [inline movement/watch_faces/sensor/minmax_face.c:4]: Please fill in your name here :)

File: movement/watch_faces/sensor/minmax_face.h [hunk 2]
```diff
@@ -0,0 +1,62 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2023 <#author_name#>
```
Comment (WesleyAC) [inline movement/watch_faces/sensor/minmax_face.h:4]: Fill out name

File: movement/watch_faces/sensor/minmax_face.h [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+#include "movement.h"
+#include "watch.h"
+
+#define LOGGING_DATA_POINTS (24)
+
+/*
+ * Log for the min. and max. temperature over the last 24h
+ *
+ * To use...
```
Comment (WesleyAC) [inline movement/watch_faces/sensor/minmax_face.h:36]: Could you add some info on how to use the face here?

**Review by WesleyAC — CHANGES_REQUESTED** [review]
Looks good overall, just a couple small things! Have you tested this on hardware?

**Comment by MarkBlyth** [comment]
> Looks good overall, just a couple small things! Have you tested this on hardware?

Yes, tested on Sensor-Watch lite (red).

**Comment by MarkBlyth** [comment]
Changes completed, thanks!

**Pull request merged**